src/document/document.ts

```diff
@@ -132,6 +132,7 @@ export interface SnapshotEvent extends BaseDocEvent {
 export interface ChangeInfo {
   message: string;
   operations: Array<OperationInfo>;
+  actor: ActorID | undefined;
```

Comment:
I think `ChangeInfo` is exposed to the user. Does `actor` need to be optional?

Reply:
The reason for optional (`actor: ActorID | undefined` is that `getActorId()` method's return type is `string | undefined`.